app/helpers/provider/admin/dashboards_helper.rb

```diff
@@ -19,17 +19,18 @@ def sign_class(widget)
     widget.percentual_change > 0 ? 'u-plus' : 'u-minus'
   end
 
-  def dashboard_collection_link(singular_name, collection, path, plural = nil, icon_name = nil)
+  def dashboard_collection_link(singular_name, collection, path, icon_name: nil, plural: nil, notice: false)
     link_to path, class: 'DashboardNavigation-link' do
       link_text = pluralize(number_to_human(collection.size), singular_name, plural)
       link_text = link_text.prepend "#{icon(icon_name)} " if icon_name
+      link_text = link_text.prepend ".u-notice" if notice
       link_text.html_safe
     end
   end
 
-  def dashboard_secondary_collection_link(singular_name, collection, path, plural = nil)
+  def dashboard_secondary_collection_link(singular_name, collection, path, icon_name: nil, plural: nil, notice: false)
```

Comment:
is there currently any call to this method that passes `notice`? where?

Reply:
There is not as far as I know. I believe this is just an enhancement rather than a refactorization.

Reply:
There are some places where we currently don't use the `dashboard_collection_link` method because this option isn't there. Once this is merged I will make use of it in another PR.